<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSettingsPanel</class>
 <widget class="ctkSettingsPanel" name="qSlicerSettingsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slicer Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="ShowToolTipsLabel">
     <property name="text">
      <string>Don't show tooltips:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="ShowToolTipsCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ShowSplashScreenLabel">
     <property name="text">
      <string>Don't show splash screen:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="ShowSplashScreenCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="FontLabel">
     <property name="text">
      <string>Font:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ctkFontButton" name="FontButton"/>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="RestoreUICheckBox">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="RestoreUILabel">
     <property name="text">
      <string>Save user interface size and position on exit:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="ConfirmExitLabel">
     <property name="text">
      <string>Confirm on exit:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="ConfirmExitCheckBox">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkFontButton</class>
   <extends>QPushButton</extends>
   <header>ctkFontButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSettingsPanel</class>
   <extends>QWidget</extends>
   <header>ctkSettingsPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
